docs/docs/dev/explore/dpp.md

```diff
@@ -1,94 +1,78 @@
 # Data plane proxy
 
-When Kuma (`kuma-cp`) runs, it waits for the data plane proxies to connect and register themselves.
+A **data plane proxy (DPP)** is the part of Kuma that runs next to each workload that is a member of the mesh.
+A DPP is composed of the following components:
 
-<center>
-<img src="/images/docs/0.4.0/diagram-10.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
-</center>
+- a `Dataplane` entity defines the configuration of the DPP
+- a `kuma-dp` binary runs on each instance that is part of the mesh. This binary spawns the following subprocesses:
```

Comment:
nitpick: this `a` looks a bit weird and the nested list does not use articles - can we omit it here as well?